Sources/Commands/init.swift

```diff
@@ -30,10 +30,25 @@ extension InitError: CustomStringConvertible {
     }
 }
 
+/// Create an initial template package.
 final class InitPackage {
+    let rootd = POSIX.getcwd()
+
+    /// The mode in use.
     let mode: InitMode
+
+    /// The name of the example package to create.
     let pkgname: String
```

Comment:
Yes, it should and I thought the same thing, I just left it out to avoid unnecessary changes in the diff.